web/frontend/angular/src/environments/environment.ts

```diff
@@ -4,6 +4,7 @@
 
 export const environment = {
     production: false,
+    baseUrl: "http://0.0.0.0:8000/",
```

Comment:
This url does not work for me. Has it been working for you? It actually wouldn't make sense if it _does_ work.

Even the simple case of visiting `http://0.0.0.0:8000` in my browser does not work, nor would I expect it to. Even if it somehow did work, our security policy would reject it because it considers it to be a different domain:

> zone.js:2863 Refused to connect to 'http://0.0.0.0:8000/api/accounts/register/' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'connect-src' was not explicitly set, so 'default-src' is used as a fallback.

To be clear, I am not suggesting that the CSP should be changed. Rather, my point is that `0.0.0.0` is not a valid host and shouldn't be used.

In a previous review of this PR, I made this suggestion: https://github.com/Comp-490-SeniorProject/site/pull/63#discussion_r836764042. Notice that it includes a diff, which is me suggesting the exact change you should make to the code. If you follow that suggestion exactly (so it won't use any `baseUrl` at all), I can confirm it will work.